scripts/teamshow

```diff
@@ -46,10 +46,8 @@ class Teamshow(object):
         """
             Get the portchannel names from minigraph.
         """
-        minigraph_path = '/etc/sonic/minigraph.xml'
-        machine_info = get_machine_info()
-        platform_info = get_platform_info(machine_info)
-        self.teams = parse_xml(minigraph_path, platform = platform_info)['PORTCHANNEL'].keys();
+        team_keys = self.db.keys(self.db.APPL_DB, PORT_CHANNEL_TABLE_PREFIX+"*")
```

Comment:
I think we should get those team_keys from CFG db instead of APP db, as we need the **expected** port channel list instead of actual list here.